geth/common/rpcall.go

```diff
@@ -0,0 +1,128 @@
+package common
+
+import (
+	gethcommon "github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+)
+
+// RPCCall represents a unit of a rpc request which is to be executed.
+type RPCCall struct {
```

Comment:
Could we also name this file as `rpccall.go`? It might be misleading as this file exports `RPCCall` struct (two `c`).